web/build_conv_mysql.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web
+
+import (
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+func buildConvMySQL(conv *internal.Conv) {
```

Comment:
The current file naming and organization of this code is somewhat confusing and makes this code harder to understand.

Usually we want to use file names that suggest groupings of functionality to make it easier to understand what code is for.

In this case, these function are test utility functions, not general code used to implement the web module.

Currently they are only used by web_test.go.

If they are unlikely to be used elsewhere, I suggest moving them to web_test.go.
If they will be used in other tests, then maybe test_util.go.
Another option could be web_test_utils.go?

I assume the motivation for splitting these functions into a separate file is because web_test.go is such a large file?  In general, while we like to keep code files to a reasonable size, we're less concerned about the size of test files -- it is usually more important to keep the test cases and their support code in one place.

Reply:
Yes Nevin, the main concern was the size of the file, i'll move it to web_test.go

Reply:
moved build_conv_postgres and build_conv_mysql functionality to web_test.go